projects/gonids/build.sh

```diff
@@ -15,20 +15,19 @@
 #
 ################################################################################
 
-# build target function
 function compile_fuzzer {
   path=$1
   function=$2
   fuzzer=$3
 
-   # Instrument all Go files relevant to this fuzzer
+  # Compile and instrument all Go files relevant to this fuzz target.
   go-fuzz -func $function -o $fuzzer.a $path
 
-   # Instrumented, compiled Go ($fuzzer.a) + fuzzing engine = fuzzer binary
-  $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer.a -lpthread -o $OUT/$fuzzer
+  # Link Go code ($fuzzer.a) with fuzzing engine to produce fuzz target binary.
+  $CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer.a -o $OUT/$fuzzer
 }
 
-compile_fuzzer /root/go/src/github.com/google/gonids/ FuzzParseRule fuzz_parserule
+compile_fuzzer github.com/google/gonids FuzzParseRule fuzz_parserule
```

Comment:
You might want to remove RUN go get github.com/google/gonids from Dockerfile.

Reply:
Why? I think that commands checks out the project to fuzz

Reply:
My bad!, skip.